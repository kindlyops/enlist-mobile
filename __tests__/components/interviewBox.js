import React from 'react'
import renderer from 'react-test-renderer'
import { fromJS } from 'immutable'
import moment from 'moment'
import InterviewBox from '../../app/components/base/interviewBox'

describe('Initial state', () => {
  test('basics', () => {
    let interview = fromJS({
      duration: 60,
      scheduledFor: moment('2019-01-01T12:00:00+00:00'),
      applicationName: 'Test Application',
      medium: 'Phone'
    })

    const tree = renderer
      .create(<InterviewBox interview={interview} />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
