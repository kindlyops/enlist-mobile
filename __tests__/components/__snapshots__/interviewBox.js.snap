// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initial state basics 1`] = `
<View
  accessible={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "borderRadius": 4,
      "marginTop": 15,
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "padding": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontWeight": "500",
        }
      }
    >
      5:30 pm to 6:30 pm 
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#848A96",
          "fontSize": 13,
        }
      }
    >
      Phone
       
      Interview with
       
      Test Application
    </Text>
  </View>
</View>
`;
